src/fides/api/ctl/database/session.py

```diff
@@ -6,13 +6,21 @@
 
 from fides.core.config import CONFIG
 
+# Parameters are hidden for security
 engine = create_async_engine(
     CONFIG.database.async_database_uri,
     echo=False,
+    hide_parameters=True,
+    logging_name="AsyncEngine",
 )
 async_session = sessionmaker(engine, class_=AsyncSession, expire_on_commit=False)
 
-sync_engine = create_engine(CONFIG.database.sync_database_uri, echo=False)
+sync_engine = create_engine(
+    CONFIG.database.sync_database_uri,
+    echo=False,
+    hide_parameters=True,
```

Comment:
I'm thinking if this should actually depend on the `dev` mode? Might be useful to still see these during development